src/Widgets/HeadlineWidget/HeadlineWidgetClass.ts

```diff
@@ -4,7 +4,7 @@ export const HeadlineWidget = provideWidgetClass('HeadlineWidget', {
   attributes: {
     alignment: ['enum', { values: ['left', 'center', 'right'] }],
     headline: 'string',
-    level: ['enum', { values: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'] }],
+    level: ['enum', { values: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'div'] }],
```

Comment:
* [ ] FYI `level: "div"` doesn't make a lot of sense. Maybe a neutral value like `none`/`text` is a better fit?
* [ ] FYI Similarly in the editing config "div" is not really something a SEO editor cares about, maybe something like "Text only" or "No semantic tag" makes more sense.